lib/rubocop/cop/rails/active_record_calculation.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Enforces the use of ActiveRecord calculation methods instead of `pluck`
+      # followed by Enumerable methods.
+      #
+      # It avoids loading potentially many values into memory by doing the
+      # calculations inside the database.
+      #
+      # @example
+      #   # bad
+      #   User.pluck(:id).max
+      #   User.pluck(:id).min
+      #   User.pluck(:age).sum
+      #
+      #   # good
+      #   User.maximum(:id)
+      #   User.minimum(:id)
+      #   User.sum(:age)
+      #
+      #   # good
+      #   User.pluck(:email).max { |email| email.length }
+      #   User.pluck(:email).max(2)
+      #   User.pluck(:id, :company_id).max
+      #   User.pluck(:age).count
```

Comment:
This is kinda a weird counterexample, because `User.count(:age)` would actually be more efficient and falls in line with this cop's premise. 🤔